ansys/mapdl/core/convert.py

```diff
@@ -341,14 +495,38 @@ def translate_line(self, line):
 
         if not line:
             return
+
+        # Cleaning ending empty arguments.
+        # Because of an extra comma added to toffst command when generating ds.dat.
+        line_ = line.split(',')[::-1] # inverting order
+        for ind, each in enumerate(line_):
+            if each:
+                break
+            else:
+                line_.pop(ind)
+        line = ','.join(line_[::-1])
+
+        #removing trailing comma
+        line = line[:-1] if line[-1] == ',' else line
+
         cmd_ = line.split(',')[0].upper()
 
+        if cmd_[:4] in ['SOLV', 'LSSO'] and self._comment_solve:
+            self.store_command('com', ["The following line has been commented on purpose:"])
+            self.store_command('com', [line])
+            return
+
+        if cmd_[:4] == '/COM':
+            # It is a comment
+            self.store_command('com', [line[5:]])
+            return
+
         if cmd_ == '*DO':
             self.start_non_interactive()
             self.store_run_command(line)
             return
 
-        if cmd_ == '*ENDDO':
+        if cmd_ == '*ENDDO' or cmd_ == '*ENDIF':
```

Comment:
This is easier to read:
```suggestion
        if cmd_ in ['*ENDDO', '*ENDIF']:
```